vosekast_control/Scale.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         self.thread_readscale = Thread()
         self.emulate = emulate
         self.is_running = False
-        self.timestamp = datetime.now()
+        self.timestamp = 0
```

Comment:
if you dont use that timestamp, you can delete that property, or?

Reply:
done